sympy/ntheory/modular.py

```diff
@@ -250,3 +250,44 @@ def combine(c1, c2):
         if symmetric:
             return symmetric_residue(n, m), m
         return n, m
+
+
+def crt_cartesian(rem, mod):
+    """
+    Crt over a cartesian product of vectors.given a vector of moduli,
```

Comment:
Please spell out "Chinese Remainder Theorem" in the first user here, in case someone doesn't know what crt stands for.